src/tools/quemap/brush.c

```diff
@@ -43,7 +43,7 @@ static void BoundBrush(bsp_brush_t *brush) {
 /**
  * @brief
  */
-static void CreateBrushWindings(bsp_brush_t *brush) {
+static void CreateBrushWindings(qbsp_brush_t *brush) {
```

Comment:
Quemap has bsp_brush_t which represents the CSG data, from what I can tell. It's then compiled down into regular bsp brushes.

The only reason this collision happened was because of the naming of the BSP stuff. I didn't want to keep the d_bsp_ prefixes (I wanted the structs to match the functions and I didn't want to have the funcs named D_Bsp_Load...), I didn't want to call them Cm either because Cm has its own BSP stuff, and map_brush_t already existed so I couldn't use that either. Any ideas would be appreciated!